examples/cms-notion/components/avatar.js

```diff
@@ -0,0 +1,12 @@
+export default function Avatar({ name, picture }) {
+  return (
+    <div className="flex items-center">
+      <img
```

Comment:
Could you start with one of the CMS examples that uses `next/image` or update this?